api/types/swarm/container.go

```diff
@@ -6,6 +6,19 @@ import (
 	"github.com/docker/docker/api/types/mount"
 )
 
+// DNSConfig specifies DNS related configurations in resolver configuration file (resolv.conf)
+// Detailed documentation is available in:
+// http://man7.org/linux/man-pages/man5/resolv.conf.5.html
+// TODO: domain is not supported yet
```

Comment:
@thaJeztah `resolv.conf` allows to specify `domain` option:

http://man7.org/linux/man-pages/man5/resolv.conf.5.html
```
domain Local domain name.
```

I will update the PR to make `domain` (not supported) explicit.